common/source/docs/common-frsky-passthrough.rst

```diff
@@ -0,0 +1,62 @@
+.. _common-frsky-passthrough:
+
+====================================
+Passthrough FrSky Telemetry Protocol
+====================================
+
+Overview
+========
+
+Passthrough FrSky Telemetry is the latest protcol available in ArduPilot and it allows the transmission of raw data directly to OpenTX without any preprocessing by OpenTX and without the need for discovery. The passthrough telemetry protocol is optimized specifically for the FrSky datalink bandwidth and enhancements include having multiple data elements in a single telemetry packet and using floating point representation (e.g. we don't really need centimeter precision altitude to be displayed on screen when flying at 100m). This results in an improved data refresh rate such that information such as attitude (roll, pitch, yaw) is displayed without any perceptible lag.
+
+Compared to regular FrSky telemetry, passthrough telemetry contains ArduPilot specific information such as flight modes, error messages, warnings, and failsafes.
+
+ArduPilot's Passthrough FrSky telemetry protocol is an open protocol that is free to use. `FlightDeck <http://www.craftandtheoryllc.com/product/flightdeck-taranis-app-for-ArduPilot-apmcopter/>`__ is an OpenTX app that uses passthrough telemetry.
+
+The ArduPilot passthrough telemetry specification including data packets and message rates is available as a spreadsheet on `Google Sheets <https://docs.google.com/spreadsheets/d/1HIfioe-EGTOtH5rOdexGlAz2E5HnCvZLm0EG6rfjNSI/edit?usp=sharing>`__.
+
+.. raw:: html
+
+  <iframe src="https://docs.google.com/spreadsheets/d/e/2PACX-1vQzfjhsUvByPERTC9vqK9v8WouYvKwZh1cQNF_KjAQLfZZiQ03nryHOn5NBwbxT3WvFw--eKEn71mz3/pubhtml?gid=1792154932&amp;single=true&amp;widget=true&amp;headers=false" width="600" height="800"></iframe>
+
+
+Required/Compatible Hardware
+============================
+
+.. image:: ../../../images/frsky_requiredhardware_flightdeck.png
+    :target: ../../../images/frsky_requiredhardware_flightdeck.png
+	
+	Common FrSky Telemetry Setup with Taranis RC transmitter running :ref:`FlightDeck <common-frsky-flightdeck>`.
+
+* An ArduPilot compatible flight controler from this list:
+	
+	- Pixhawk
+	- Pixhawk derivatives (clones)
+	- Pixhawk 2.1
+	- PixRacer
+	- Pixhack
+	- (Linux Boards and APM2.6 are not supported at this time)
```

Comment:
Agreed. It may not be tested on Linux boards yet, but I don't see any reason why it wouldn't work. Getting it working on one Linux board as a test would allow you to say you have Linux board support. That's huge.